### ISSUE TITLE
[BUG]: Guys. Desperate to roll this out and documentation is not at all clear.  Please help !

### ISSUE THREAD

**Issue by drachar** [microsoft/powerapps-tools#162]
**Describe the bug**
Followed instructions and my app is nothing but a blank purple screen.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Interface(please complete the following information):**
 - Version date[e.g. 2020.04.03]

**Smartphone (please complete the following information):**

**Additional context**

**Comment by manuelap-msft** [comment]
Hello! What part of the documentation are you struggling with? Is it the setup instructions or the more general documentation?

**Comment by drachar** [comment]
Hi

 

I followed the instructions to get the Crisis Management up and loaded.  Set up all the Sharepoint site details and, I think, I got all the connections set up correctly.  But when I load the App the home screen is blank.

 

I am looking for some form of troubleshooting advice or documentation – as in how to find the thing I missed rather than starting the whole exercise from the beginning again.

 

Thanks

Richard
